core/src/broadcast_stage/standard_broadcast_run.rs

```diff
@@ -300,13 +278,7 @@ impl StandardBroadcastRun {
         socket_sender.send((data_shreds.clone(), batch_info.clone()))?;
```

Comment:
you can merge the data and coding shreds here rather than sending them separately.

Reply:
I have some follow up changes here. I will revisit this part.
But for now I lean to not make this patch larger.